developer-docs/installation/sync_user_org.md

```diff
@@ -0,0 +1,112 @@
+
+````
+Sync_run.sh --context_param sunbird_cassandra_server="{sunbird_cassandra_server}" --context_param sunbird_cassandra_port="{sunbird_cassandra_port}" --context_param sunbird_cassandra_username="{sunbird_cassandra_username}" --context_param sunbird_cassandra_password="{sunbird_cassandra_password}" --context_param sunbird_sync_api_endpoint="{sunbird_sync_api_endpoint}" --context_param sunbird_sync_api_key="{sunbird_sync_api_key}" --context_param sunbird_sync_block_size="{sunbird_sync_block_size}" --context_param sunbird_sync_sleep_time="{sunbird_sync_sleep_time}" --context_param sunbird_cassandra_table="organisation" --context_param sunbird_sync_object_type="organisation" --context_param dry_run=false
+````
+
+3. Run the following command to sync **specific users** data.
+
+````
+Sync_run.sh --context_param sunbird_cassandra_server="{sunbird_cassandra_server}" --context_param sunbird_cassandra_port="{sunbird_cassandra_port}" --context_param sunbird_cassandra_username="{sunbird_cassandra_username}" --context_param sunbird_cassandra_password="{sunbird_cassandra_password}" --context_param sunbird_sync_api_endpoint="{sunbird_sync_api_endpoint}" --context_param sunbird_sync_api_key="{sunbird_sync_api_key}" --context_param sunbird_sync_block_size="{sunbird_sync_block_size}" --context_param sunbird_sync_sleep_time="{sunbird_sync_sleep_time}" --context_param sunbird_cassandra_table="user" --context_param sunbird_sync_object_type="user" --context_param sunbird_sync_id_file="{sunbird_sync_id_file}" --context_param dry_run=false
+````
+
+Run the following command to sync **specific organisations** data.
+
+````
+Sync_run.sh --context_param sunbird_cassandra_server="{sunbird_cassandra_server}" --context_param sunbird_cassandra_port="{sunbird_cassandra_port}" --context_param sunbird_cassandra_username="{sunbird_cassandra_username}" --context_param sunbird_cassandra_password="{sunbird_cassandra_password}" --context_param sunbird_sync_api_endpoint="{sunbird_sync_api_endpoint}" --context_param sunbird_sync_api_key="{sunbird_sync_api_key}" --context_param sunbird_sync_block_size="{sunbird_sync_block_size}" --context_param sunbird_sync_sleep_time="{sunbird_sync_sleep_time}" --context_param sunbird_cassandra_table="organisation" --context_param sunbird_sync_object_type="organisation" --context_param sunbird_sync_id_file="{sunbird_sync_id_file}" --context_param dry_run=false
+````
+
+4. On completion, refer to the success and failure logs that are generated 
+
+5. When running the script for specified ids, give all the ids as below format in the csv file and no headers required.
+Example:
+3150aa06-333e-4d6d-8d81-3d8e14e7b245
+a354f4c4-a5a7-4eee-9b69-7305cc52a480
+bf1efbd7-4ad6-4e2f-925d-0a6095aeafb0
+54660e04-4d7c-4f12-b307-c5a7174b68d4
+3b87a8a1-0831-453c-8230-b5306daca44f
+b6de8b58-144e-49b2-a65a-835cf81ee700
+
+6. Another way proposed for running failure Id's in the failure log is by running the script

```

Comment:
Add an instruction to fill the variables like URL and Bearer in the script before running, 
URL: https://DNS
Bearer: value